docs/intro.md

```diff
@@ -5,7 +5,9 @@
 # Introduction
 
 This is the manual for msprime, a population genetics simulator
-based on [tskit](https://tskit.dev). Msprime can simulate
```

Comment:
This reads strangely to me - "trivial" isn't a positive word IMO.

How about we make the first sentence "This is the manual for msprime, a population genetics simulator of ancestry and DNA sequence evolution based on tskit"

Reply:
Yes, I agree about "trivial", but couldn't find the right wording. Yours sounds good.

Reply:
OK, let's change then.

Reply:
Do you want to or shall I?

Reply:
Can you do it please